androguard/core/bytecodes/apk.py

```diff
@@ -767,25 +778,6 @@ def is_multidex(self):
         dexre = re.compile("^classes(\d+)?.dex$")
         return len([instance for instance in self.get_files() if dexre.search(instance)]) > 1
 
-    def get_elements(self, tag_name, attribute, with_namespace=True):
```

Comment:
I would not remove them, as we saw in the past that some functions are still used...
they will still work fine, right?